backend/tracim_backend/views/core_api/schemas.py

```diff
@@ -1026,6 +1033,11 @@ class WorkspaceMinimalSchema(marshmallow.Schema):
     workspace_id = marshmallow.fields.Int(example=4, validate=strictly_positive_int_validator)
     slug = StrippedString(example="intranet")
     label = StrippedString(example="Intranet")
+    access_type = StrippedString(
+        example=WorkspaceAccessType.CONFIDENTIAL.value,
+        validate=workspace_access_type_validator,
+        default=WorkspaceAccessType.CONFIDENTIAL.value,
```

Comment:
I would rather not leave a default here, the access type is as important as the label.

Reply:
i understand and this is not compilant with specification, see https://github.com/tracim/tracim/issues/3536#issuecomment-692596230 but such change break frontend, so if we are strict here, we do need to fix this in this PR or in a very soon PR, because this break frontend.

Reply:
fixed.